pipeline-model-definition/src/main/resources/org/jenkinsci/plugins/pipeline/modeldefinition/ClosureModelTranslator.groovy

```diff
@@ -47,12 +47,12 @@ public class ClosureModelTranslator implements MethodMissingWrapper, Serializabl
     /**
      * Placeholder to make sure 'agent none' works.
      */
-    boolean none = false
+    String none = "none"
 
     /**
      * Placeholder to make sure 'agent any' works.
      */
-    boolean any = true
+    String any = "any"
```

Comment:
Annoyingly, they are still being used - otherwise `agent none` and `agent any` won't work.